client/go/auth0/auth0.go

```diff
@@ -129,6 +129,7 @@ func (a *Auth0) IsLoggedIn() bool {
 	}
 
 	// Check if token is valid.
+	// TODO: Choose issuer based on system
```

Comment:
This can be deduced Auth0 device flow config from api endpoint (https://api.vespa-external-cd.aws.oath.cloud:4443/auth0/v1/device-flow-config).